acbuild/write.go

```diff
@@ -33,7 +34,8 @@ var (
 func init() {
 	cmdAcbuild.AddCommand(cmdWrite)
 
-	cmdWrite.Flags().BoolVar(&overwrite, "overwrite", false, "overwrite output ACI")
+	cmdWrite.Flags().BoolVar(&overwrite, "overwrite", false, "overwrite the resulting ACI")
+	cmdWrite.Flags().BoolVar(&compress, "compress", false, "compress the resulting ACI")
```

Comment:
how about `--no-compression` like actool?
Otherwise I'm happy to just omit this until we have a use case - at that time it'll take five minutes to add.